google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITGrpcTest.java

```diff
@@ -156,4 +188,53 @@ public void deleteHmacKey() {
     storageFixture.getInstance().updateHmacKeyState(hmacKey.getMetadata(), HmacKeyState.INACTIVE);
     storageFixture.getInstance().deleteHmacKey(hmacKey.getMetadata());
   }
+
+  @Test
+  public void object_writeGetRead() {
```

Comment:
Are you using snakecase to clearly group test names while the refactoring is on-going?

Reply:
Not for the refactoring, simply to group the conceptually related tests.